lib/lambda_ethereum_consensus/logger/console_logger.ex

```diff
@@ -0,0 +1,46 @@
+defmodule ConsoleLogger do
+  @moduledoc """
+  Custom logger formatter for console output.
+  """
+
+  def format(level, message, timestamp, metadata) do
+    pattern = Logger.Formatter.compile(" $time $message ")
```

Comment:
You could move this to a `@` module constant. That way it's only run at compile time.